src/default_env.tres

```diff
@@ -1,19 +1,10 @@
 [gd_resource type="Environment" load_steps=2 format=2]
 
 [sub_resource type="ProceduralSky" id=1]
-radiance_size = 0
 sky_top_color = Color( 0, 0.00392157, 0.00392157, 1 )
-sky_horizon_color = Color( 0.109804, 0.129412, 0.207843, 1 )
-sky_curve = 0.131768
-sky_energy = 0.78
-ground_bottom_color = Color( 0.145098, 0.227451, 0.0470588, 1 )
-ground_horizon_color = Color( 0.0392157, 0.0509804, 0.0470588, 1 )
-sun_color = Color( 0.0862745, 0, 0, 1 )
-sun_latitude = -29.33
-sun_longitude = -41.98
-sun_angle_min = 53.41
-sun_angle_max = 90.83
-texture_size = 1
+sky_horizon_color = Color( 0.160784, 0.109804, 0.207843, 1 )
```

Comment:
Just wondering, why were the default_env values for the sky changed? Could you please add a before/after if this is intentional? Just wondering as this was edited for the study room + for the garden rooms previously

Reply:
hmm not sure why this was changed I thought I removed this from the commit. Theres no need for this change for the work here.